static/js/people.js

```diff
@@ -134,7 +137,7 @@ exports.huddle_string = function (message) {
     }
 
     let user_ids = _.map(message.display_recipient, function (recip) {
-        return recip.id;
+        return parseInt(recip.id, 10);
```

Comment:
`.id` seems like it ought to be a `number` already? If it isn’t, we should fix that.

Reply:
fixed, thanks